timeline.tex

```diff
@@ -13,17 +13,22 @@ \section{Roadmap and Timeline} \label{sec:timeline}
 
 Milestone dates are given as min-max ranges to indicate the associated uncertainty. 
 Typically the near date corresponds to the current Project forecast, plus any additional operational uncertainty.
-The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty and cannot be surpassed without the Project re-baselining its schedule.
+The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty.
 An intermediate (typically mid-range) date is used by the Rubin Operations teams for planning purposes. 
 
-The LSST survey will start shortly after the completion of the SV surveys, currently expected to be sometime between February 2025 and September 2025.
+The LSST survey start is currently expected to be sometime between February 2025 and September 2025.
 The timing of the Commissioning observations is somewhat less uncertain and the timing of the release of those data to the community can be projected to within a few months at the time of writing.
 
 Table \ref{tab:timeline} shows the nominal date ranges for the various elements of the Early Science Program. 
+The shaded region contains roughly 80% of the probability, while the lefthand edge of the shaded range indicates the earliest date the milestone could be reached. 
+The darker regions give a very rough indication of the +/-1 sigma error bars. 
+Over the course of the commissioning period, we expect these shaded regions to shrink as our understanding of the remaining schedule uncertainty improves. 
+However, there is still the possibility of the assumptions underlying these distributions being wrong: this is just our best estimate at the current time.
+
 The next key milestone in the Early Science Program is the release of DP0.3. 
 Scheduled for mid-2023, DP0.3 will be the last in the DP0 series, which is based on simulated LSST-like data. 
-The late dates for the DP2 and DR1 milestones allow for the possibility that the Project completes within its late date, but in doing so reduces the amount of on-sky LSSTCam commissioning time.
-In this eventuality, the operations team would spend up to 3 months prior to commencing the 10-year LSST survey completing any remaining SV Survey observations , such that DP2 could be realized as planned.
+The late dates for the DP2 and DR1 milestones allow for the possibility that the Project completes within its late date, but in doing so spends less time on-sky with LSSTCam.
+In this eventuality, the operations team would spend up to 2 months prior to commencing the 10-year LSST survey completing any remaining observations such that DP2 could be realized as planned.
```

Comment:
I would prefer to not say "complete any remaining observations" to avoid any possible misinterpretation that ops is finishing the work of construction. I prefer " .....prior to commencing the 10-year survey collecting more on-sky data to complement and extend the datasets collected during commissioning